src/Console/Command/CommandHelp.php

```diff
@@ -420,7 +430,8 @@ function (FixerOptionInterface $optionA, FixerOptionInterface $optionB) {
             }
         }
 
-        return $help;
+        // prevent "\</foo>" from being rendered as an escaped literal style tag
+        return preg_replace('#\\\\(</.*?>)#', '<<$1', $help);
```

Comment:
This happens with `native_function_invocation` description: _Add leading `\` before function invocation [...]._ You may make it more obvious by replacing `<info>` tags with `<error>` tags.